docs/self-managed/components/modeler/web-modeler/configuration/configuration.md

```diff
@@ -183,6 +183,174 @@ Web Modeler supports syncing files via [Git Sync](../../../../../components/mode
 | Azure DevOps  | `CAMUNDA_MODELER_GITSYNC_AZURE_AUTHORITY_BASE_PATH` | URL used to access authentication and authorization services for Microsoft cloud identities.                                  | `https://login.microsoftonline.com`          |
 | Bitbucket     | `CAMUNDA_MODELER_GITSYNC_BITBUCKET_BASEURL`         | The base URL of Bitbucket Cloud.                                                                                              | `https://api.bitbucket.org/2.0/repositories` |
 
+### Copilot
```

Comment:
💭 The configuration in here is quite extensive - I'm wondering if it's worth introducing a new sub-page similar to docs.camunda.io/docs/next/self-managed/components/modeler/web-modeler/configuration/database/